llvm/test/Analysis/CostModel/RISCV/reduce-add.ll

```diff
@@ -1,8 +1,10 @@
 ; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py
-; RUN: opt < %s -mtriple=riscv32 -mattr=+v -passes="print<cost-model>" -cost-kind=throughput 2>&1 -disable-output | FileCheck %s
-; RUN: opt < %s -mtriple=riscv64 -mattr=+v -passes="print<cost-model>" -cost-kind=throughput 2>&1 -disable-output | FileCheck %s
-; RUN: opt < %s -mtriple=riscv32 -mattr=+v -passes="print<cost-model>" -cost-kind=code-size 2>&1 -disable-output | FileCheck %s --check-prefix=SIZE
-; RUN: opt < %s -mtriple=riscv64 -mattr=+v -passes="print<cost-model>" -cost-kind=code-size 2>&1 -disable-output | FileCheck %s --check-prefix=SIZE
+; RUN: opt < %s -mtriple=riscv32 -mattr=+v -passes="print<cost-model>" -cost-kind=throughput 2>&1 -disable-output --type-based-intrinsic-cost=true | FileCheck %s
```

Comment:
Please revert the change (in this and other files) to add --type-based-intrinsic-cost=true.

Reply:
You mean remove `--type-based-intrinsic-cost=true` from the original `RUN` and add another version of the `RUN` line with `--type-based-intrinsic-cost=true`?  Or you thought we have better approach to test the instruction cost of vp-intrinsics here?
I am not  sure what you want?

Reply:
You modified the run lines, please remove the modification in this change.  If you want to modify them, do that in it's own change.  (I am skeptical of your motivation for that separate change.)

Reply:
Reverted. 
Add new run lines for the `type-based` queries and check the instruction costs are same as normal query.